src/commcare_cloud/ansible/roles/citusdb/meta/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+dependencies:
+  - role: monit
+  - role: postgresql_base
+    postgresql_version: '11'
+    postgresql_process_path: /var/run/postgresql
+    postgresql_cluster_name: '11/main'
+    postgresql_shared_preload_libraries:
+     - 'citus'
```

Comment:
this will be added before citus is installed. Will that prevent PG from starting?

Reply:
I tried is to set it after installing citus, but than PostgreSQL had to be restart. It was annoying since everytime I run the playbook it was restarting/reloading as the base role was setting it empty and `citus role` was again setting it to `citus`
I didn't find any trouble while testing. It could also be because I already had citus installed in test machines.

Reply:
ok